.gitignore

```diff
@@ -17,6 +18,10 @@
 /static/**/*.svg
 
 ### Node ###
+
+# lock files
+package-lock.json
```

Comment:
this need to be removed from gitignore file as `package-lock.json` is intended to be checked in.